src/Internal.AspNetCore.BuildTools.Tasks/GetDotNetHost.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#if NETSTANDARD1_6
+
+using System;
+using System.Runtime.InteropServices;
+using System.IO;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Microsoft.AspNetCore.BuildTools
+{
+    /// <summary>
+    /// Uses the current runtime information to find the "dotnet.exe" path.
+    /// This requires that MSBuild itself execute on top of Microsoft.NETCore.App.
+    /// </summary>
+    public class GetDotNetHost : Task
+    {
+        private const string MuxerName = "dotnet";
+
+        [Output]
+        public string MuxerPath { get; set; }
```

Comment:
Don't know what a Muxer is. I would find a way simpler name like ExecutablePath or DotNetExecutable